api/go-internal/localrecord.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
+
+package tinder;
+
+import "github.com/gogo/protobuf/gogoproto/gogo.proto";
+
+option go_package = "berty.tech/berty/go/internal/tinder";
+
+option (gogoproto.goproto_enum_prefix_all) = false;
+option (gogoproto.marshaler_all) = true;
+option (gogoproto.unmarshaler_all) = true;
+option (gogoproto.sizer_all) = true;
+
+message LocalRecord {
+    message Record {
+      string cid = 1;
+      int64 expire = 2;
+    }
+    repeated Record records = 1;
+}
```

Comment:
Maybe it's normal, but it's very strange for me to "one `LocalRecord` that contains many `Records`,

shouldn't it be `LocalRecords` instead?

---

btw, you can get cleaner generated code like this:

```suggestion
message LocalRecord {
    repeated Entry records = 1;
}

message Entry {
  string cid = 1;
  int64 expire = 2;
}
```

the generated code will change from:

```go
 	lr := &LocalRecord{Records: []*LocalRecord_Record{}} 
```

to

```go
 	lr := &LocalRecord{Records: []*Entry{}} 
```

---

buf if things are not voluntarily singular here, I think the best is just:

```suggestion
message Records {
    repeated Record records = 1;
}

message Record {
  string cid = 1;
  int64 expire = 2;
}
```

Reply:
ok, I've chosen the second solution to be more generic. Thanks!